WBooks/Book_Information/BookInformationViewController.swift

```diff
@@ -9,8 +9,40 @@
 import UIKit
 import ReactiveSwift
 
+enum BookStatus: String {
+    case avaliable = "Avaliable"
+    case notAvaliable = "Not Avaliable"
+    
+    var textColor: UIColor {
```

Comment:
`BookStatus` is like a business model, small but it still business logic. So we could move it to Models with Book or Rent or its own file.
This `textColor` property is only used in `BookInformationView.swift`, and `rentEnabled` just here, so we prefer to put it there, because it isn't something all the project needs to know about the BookStatus, just that view or this controller.

So we would have:
```
[BookStatus.swift]
enum BookStatus: String {
   case ...
}
[BookInfoController.swift]
fileprivate extension BookStatus {
    var rentEnabled: Bool { ... }
}
[BookInfoView.swift]
fileprivate extension BookStatus {
    var textColor: UIColor { ... }
}
```

We love how we can use extensions in swift 😍